docs/source/testing_roles.rst

```diff
@@ -35,6 +35,60 @@ be sufficient for the `molecule test` command to work.
 
 Alternatively you can test roles in with :ref:`AnsibleEE<testing with ansibleee>`.
 
+Creating a VM to run molecule
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The
+[edpm_libvirt](https://github.com/openstack-k8s-operators/edpm-ansible/tree/main/roles/edpm_libvirt)
+molecule senario uses the delegated driver to run against localhost.
+Thus, it's better to run it inside an ephemeral virtual machine as
+it cannot be run insider of a container.
+
+Boot a VM based on `CentOS-Stream-GenericCloud-9-latest.x86_64.qcow2`
+from https://cloud.centos.org/centos/9-stream/x86_64/images.
+
+Use [repo-setup](https://github.com/openstack-k8s-operators/repo-setup)
+to configure the DNF repositories of the VM.
+
+.. code-block:: console
+
+    pushd /tmp
+    curl -sL https://github.com/openstack-k8s-operators/repo-setup/archive/refs/heads/main.tar.gz | tar -xz
+    pushd repo-setup-main
+    python3 -m venv ./venv
+    PBR_VERSION=0.0.0 ./venv/bin/pip install ./
+    ./venv/bin/repo-setup current-podified-dev
+    popd
+
+Clone the git repository on the VM.
+
+.. code-block:: console
+
+    dnf install -y git
+    git clone git@github.com:openstack-k8s-operators/edpm-ansible.git
```

Comment:
nit if you follow this as written we are still in 
/tmp/repo-setup-main
so maybe we should add a cd ~ or cd .. but that not really that imporant

Reply:
OK, i added `cd ~`